dev/com.ibm.ws.security.acme/resources/com/ibm/ws/security/acme/resources/AcmeMessages.nlsprops

```diff
@@ -206,3 +206,23 @@ CWPKI2044E.useraction=Ensure that all the certificates in the certificate chain
 CWPKI2045W=CWPKI2045W: The certificate with {0} serial number that is signed by the ACME certificate authority at the {1} URI is not valid until {2}.
 CWPKI2045W.explanation=The valid period on the certificate is in the future. SSL/TLS requests fail until the current date and time are within the range that is specified by the valid period on the certificate.
 CWPKI2045W.useraction=Update the local time on the server if the time is incorrect.
+
+CWPKI2046E=CWPKI2046E: Failed to generate a new account key pair while refreshing the account key pair. The error is ''{0}''.
+CWPKI2046E.explanation=Could not generate a new account key pair.
+CWPKI2046E.useraction=Ensure that the directory that contains the existing account key pair file is writable. Review the error message for details on the failure.
+
+CWPKI2047E=CWPKI2047E: Failed to register the new account key pair with the ACME certificate authority. The error is ''{0}''.
+CWPKI2047E.explanation=The ACME certificate authority returned an error while updating the account key pair.
+CWPKI2047E.useraction=Ensure that the existing account key pair is valid. Review the error message for details on the failure. 
+
+CWPKI2048I=CWPKI2048I: Successfully refreshed the account key pair. The old account key pair has been backed up to the {0} file.
```

Comment:
Change "has been" to "was" or change the phrase to "backup is the {0} file.

Reply:
Agreed.